Add the expected overhead for the .NET profiler (#14862)
DataDog/documentation: content/en/tracing/profiler/profiler_troubleshooting.md

- Application type (for example, Web application running in IIS).

## High CPU usage when enabling the profiler

The profiler has a fixed overhead. The exact value can vary but should be expected to be about:
 -  200ms of CPU time per second on Linux (0.2 CPU)
 -  20ms of CPU time per second on Windows (0.02 CPU)

> This fixed cost means that the relative overhead of the profiler can be significant in very small containers. For instance, if running in a Linux container with 0.4 CPU assigned, the fixed cost of 0.2 CPU means that the relative overhead will be of 50%. It is recommended to adjust the container limits accordingly.

Reviewer: ```suggestion
This fixed cost means that the relative overhead of the profiler can be significant in very small containers. For example, if you run the profiler in a Linux container with 0.4 CPU assigned, the fixed cost of 0.2 CPU means that the relative overhead is 50%. Adjust the container limits accordingly.
```
